rest/config_test.go

```diff
@@ -142,6 +142,26 @@ func TestConfigValidationDeltaSync(t *testing.T) {
 	}
 }
 
+func TestConfigValidationImportPartitions(t *testing.T) {
```

Comment:
Looks like we have most of this already in `TestConfigValidationImport`, but it's not doing the CE/EE checks that this test is doing.

Perhaps worth enhancing that test instead of adding this one?

```
--- FAIL: TestConfigValidationImport (0.00s)
    --- FAIL: TestConfigValidationImport/Import_partitions_set,_shared_bucket_not_enabled (0.00s)
        require.go:738:
            	Error Trace:	config_test.go:257
            	Error:      	"[]" should have 1 item(s), but has 0
            	Test:       	TestConfigValidationImport/Import_partitions_set,_shared_bucket_not_enabled
    --- FAIL: TestConfigValidationImport/Import_disabled,_but_partitions_set (0.00s)
        require.go:738:
            	Error Trace:	config_test.go:257
            	Error:      	"[]" should have 1 item(s), but has 0
            	Test:       	TestConfigValidationImport/Import_disabled,_but_partitions_set
    --- FAIL: TestConfigValidationImport/Too_many_partitions (0.00s)
        require.go:738:
            	Error Trace:	config_test.go:257
            	Error:      	"[]" should have 1 item(s), but has 0
            	Test:       	TestConfigValidationImport/Too_many_partitions
    --- FAIL: TestConfigValidationImport/Not_enough_partitions (0.00s)
        require.go:738:
            	Error Trace:	config_test.go:257
            	Error:      	"[]" should have 1 item(s), but has 0
            	Test:       	TestConfigValidationImport/Not_enough_partitions
```

Reply:
Moving the new test in here didn't quite fit, so left it as-is and marked this one EE only.